sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSourceSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.sqs.scaladsl
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.sqs.SqsSourceSettings
+import akka.stream.scaladsl.Sink
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.services.sqs.AmazonSQSAsyncClient
+import org.elasticmq.rest.sqs.{SQSRestServer, SQSRestServerBuilder}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpec}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.util.{Random, Try}
+
+class SqsSourceSpec extends WordSpec with BeforeAndAfterAll with ScalaFutures with Matchers {
```

Comment:
Instead of using `WordSpec` with `Await.ready/result` I suggest you switch over to `AsyncWordSpec` which has been introduced in ScalaTest 3.